contracts/CampaignFactory.sol

```diff
@@ -108,39 +143,112 @@ contract CampaignFactory {
             }
         }
         orgCampaigns[organisation].pop();
-        emit campaignEnded(organisation, address(campaign));
+        emit CampaignEnded(organisation, address(campaign));
     }
 
-    // withdraw commission
+    /**
+     * @notice Allows the manager of this CampaignFactory contract to take out all collected commissions
+     * to date
+     */
     function withdrawCommissions() public ownerOnly {
         require(address(this).balance > 0, "No remaining balance");
         address payable ownerAddr = address(uint160(owner));
         uint256 amount = address(this).balance;
 
         ownerAddr.transfer(amount);
-        emit commissionWithdrawn(amount);
+        emit CommissionWithdrawn(amount);
     }
 
+    /**
+     * @notice Deletes an untrustworthy beneficiary from this contract's orgCampaigns mapping
+     * @dev This does not affect the status of the beneficiary in the IAM contract
+     * @param organisation The address of the beneficiary that will be deleted from orgCampaigns
+     */
     function deleteDistrustedOrg(address organisation) public ownerOnly {
         require(isDistrust(organisation) == true, "Organisation status is not distrust");
-        require(block.timestamp >= IAMContract.getRefundPeriod(organisation) + secondsInSixMonths(), "Refund period is ongoing");
+        require(block.timestamp >= IAMContract.getRefundPeriod(organisation) + getSecondsInSixMonths(), "Refund period is ongoing");
         uint256 len = orgCampaigns[organisation].length - 1;
         for (int i = int(len); i >= 0; i--) {
             Campaign c = Campaign(orgCampaigns[organisation][uint256(i)]);
             c.returnRemainingBalance();
             deleteCampaignFromMapping(organisation, c);
         }
         deleteOrgFromMapping(organisation);
-        emit refundComplete(organisation);
+        emit RefundComplete(organisation);
     }
 
+    /**
+     * @notice Checks if a beneficiary has the 'Verified' status
+     * @param organisation The address representing the beneficiary to check
+     * @return true if the beneficiary has a 'Verified' status, false otherwise
+     */
+    function isVerified(address organisation) public view returns (bool) {
+        return IAMContract.isVerified(organisation);
+    }
```

Comment:
Likewise as above, most `views` don't need docstrings. Simple setters and getters also. Applies to some of the methods below.

Reply:
Okay, documentations for all functions after this function (inclusive) have been removed except for `deleteCampaignFromMapping` and `deleteOrgFromMapping` functions in e8c988b